src/GafferSceneUI/AttributeInspector.cpp

```diff
@@ -248,18 +248,36 @@ IECore::ConstObjectPtr AttributeInspector::value( const GafferScene::SceneAlgo::
 	return nullptr;
 }
 
-IECore::ConstObjectPtr AttributeInspector::fallbackValue( const GafferScene::SceneAlgo::History *history ) const
+IECore::ConstObjectPtr AttributeInspector::fallbackValue( const GafferScene::SceneAlgo::History *history, std::string &description ) const
 {
 	if( const auto inheritedAttribute = history->scene->fullAttributes( history->context->get<ScenePlug::ScenePath>( ScenePlug::scenePathContextName ) )->member( m_attribute ) )
 	{
+		ScenePlug::PathScope pathScope( history->context.get() );
+		pathScope.setCanceller( Context::current()->canceller() );
```

Comment:
The `value()` method is documented as being called with `history->context()` already scoped. I think we should document the same for `fallbackValue()` and rely on that here (it seems to be true already, from a glance at the code). I don't think we want inspectors to be responsible for canceller juggling.

I _think_ this means that we can just do `PathScope pathScope( Context::current() )` here.

Reply:
Improved the fallbackValue documentation as part of the move under `value()` in cbb151a.

Switched to `PathScope pathScope( Context::current() )` in 8a2d9d8.